components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/connector/AdminForcedPasswordResetConfigImplTest.java

```diff
@@ -0,0 +1,181 @@
+
+        for (int i = 0; i < properties.size(); i++) {
+            assertEquals(propertiesList[i], propertiesExpected[i]);
+        }
+    }
+
+    @DataProvider(name = "provideDefaultPropertyValues")
+    public Object[][] providePropertyValues() {
+        String tenantDomain_1;
+        String tenantDoamin_2;
+
+        tenantDomain_1 = null;
+        tenantDoamin_2 = "admin";
+
+        return new Object[][]{
+                {tenantDomain_1}, {tenantDoamin_2}
+        };
+    }
+
+    /**
+     * @param tenatDomain
+     * @throws IdentityGovernanceException
+     */
+    @Test(dataProvider = "provideDefaultPropertyValues")
+    public void testGetDefaultPropertyValues(String tenatDomain) throws IdentityGovernanceException {
+        String testEnableAdminPasswordResetWithRecoveryLink = "false";
+        String testEnableAdminPasswordResetWithOTP = "false";
+        String testEnableAdminPasswordResetOffline = "false";
+
+        Map<String, String> defaultProperties = new HashMap<>();
+        defaultProperties.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_WITH_RECOVERY_LINK,
+                testEnableAdminPasswordResetWithRecoveryLink);
+        defaultProperties.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_WITH_OTP,
+                testEnableAdminPasswordResetWithOTP);
+        defaultProperties.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_OFFLINE,
+                testEnableAdminPasswordResetOffline);
+
+        Properties propertiesExpected = adminForcedPasswordResetConfigIml.getDefaultPropertyValues(tenatDomain);
+        Map<String, String> defaultPropertiesExpected = new HashMap<String, String>((Map) propertiesExpected);
+        Assert.assertEquals(defaultProperties, defaultPropertiesExpected, "Maps are equal");
+    }
+
+    @DataProvider(name = "defaultPropertyValues1")
+    public Object[][] provideDefaultPropertyValues1() {
+        String tenantDomain1;
+        String tenantDomain2;
+
+        tenantDomain1 = "null";
+        tenantDomain2 = "admin";
+
+        String[] propertyNames1;
+        String[] propertyNames2;
+
+        propertyNames1 = null;
+        propertyNames2 = new String[]{"property1", "property2", "property3"};
+        return new Object[][]{
+                {propertyNames1, tenantDomain1}, {propertyNames2, tenantDomain2}
+        };
+    }
+
+    /**
+     * @param propertyNames
+     * @param tenantDomain
+     * @throws IdentityGovernanceException
+     */
+    @Test(dataProvider = "defaultPropertyValues")
+    public void testGetDefaultPropertyValues(String[] propertyNames, String tenantDomain) throws IdentityGovernanceException {
+        Map<String, String> defaultPropertyValuesExpected = adminForcedPasswordResetConfigIml.getDefaultPropertyValues(propertyNames, tenantDomain);
+        assertEquals(null, defaultPropertyValuesExpected);
```

Comment:
Swap the parameters. In assertEquals method, first parameter should be the result and the second should be the expected.